[2025-02-08T19:00:02.190+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:00:02.212+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2025-02-08T19:00:02.222+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2025-02-08T19:00:02.222+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:00:02.243+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-09-18 00:00:00+00:00
[2025-02-08T19:00:02.287+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=483) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:00:02.269+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-09-18T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmps5r2mwrv']
[2025-02-08T19:00:02.289+0000] {standard_task_runner.py:72} INFO - Started process 515 to run task
[2025-02-08T19:00:02.301+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask generate_account_dim_data
[2025-02-08T19:00:02.546+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-09-18T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:00:02.816+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-18T00:00:00+00:00'
[2025-02-08T19:00:02.819+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:00:02.889+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T19:00:02.890+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:00:02.903+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:00:02.903+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-09-18T00:00:00+00:00, execution_date=20240918T000000, start_date=20250208T190002, end_date=20250208T190002
[2025-02-08T19:00:02.938+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:00:02.958+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:00:02.961+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:39:38.079+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:39:38.154+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2025-02-08T20:39:38.213+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2025-02-08T20:39:38.214+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:39:38.274+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-09-18 00:00:00+00:00
[2025-02-08T20:39:38.302+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=577) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:39:38.303+0000] {standard_task_runner.py:72} INFO - Started process 676 to run task
[2025-02-08T20:39:38.306+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-09-18T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpypu3p_ij']
[2025-02-08T20:39:38.314+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask generate_account_dim_data
[2025-02-08T20:39:38.506+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-09-18T00:00:00+00:00 [running]> on host 7e959c1a2a8b
[2025-02-08T20:39:39.138+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-18T00:00:00+00:00'
[2025-02-08T20:39:39.142+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:39:40.518+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T20:39:40.520+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:39:40.534+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:39:40.536+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-09-18T00:00:00+00:00, execution_date=20240918T000000, start_date=20250208T203938, end_date=20250208T203940
[2025-02-08T20:39:40.811+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:39:40.841+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:39:40.863+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
